pkg/util/fargate/hostname_core.go

```diff
@@ -0,0 +1,14 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build !fargateprocess
+
+package fargate
+
+// GetFargateHost returns the Fargate hostname used
+// by the core Agent for Fargate
+func GetFargateHost() (string, error) {
+	return "", nil
```

Comment:
shouldn't we return an error here ?

Reply:
no, it should be an empty string

Reply:
So if we're running on fargate we want to set the hostname to an empty string ?

Reply:
yes, we're doing this already https://github.com/DataDog/datadog-agent/blob/master/pkg/util/hostname.go#L161

Reply:
ok, I didn't knew !